tests/test_helpers.py

```diff
@@ -912,14 +912,16 @@ def test_safe_join(self):
             # Base directory is always considered safe
             (("../", "a/b/c"), "../a/b/c"),
             (("/..",), "/.."),
-        )
-
-        for args, expected in passing:
-            assert flask.safe_join(*args) == expected
+        ),
+    )
+    def test_safe_join(self, passing):
+        args = passing[0]
```

Comment:
Split this into multiple parameters. `("args", "expected")` instead of `"passing"`.

Reply:
I updated them to args and expected as requested.